FannyAndersson/PayWay
# Feature/side menu

Hello!
Welcome our side-menu! It is created as SideNav component from react-materialize but it uses also a handmade component SideMenuItem with customized markup and onClick method.
How to test:

- Be sure that user logged in
- Click on menu button - Side-menu slide in on the left
- Click on gray overlay - Side-menu slide out
- Click on menu button again - click on any menu-item - check path - it must be changed. Menu-item got class Active and Side-menu slide out. Open it once again - menu-item has accent color (purple).
- Update page - open Side-menu again - active menu-item has accent color.

NOTE: routes in the menu-items links can be changed if it needs =)
